### ISSUE TITLE
Timezone in Container is always UTC

### ISSUE THREAD

**Issue by tinyoverflow** [pterodactyl/panel#2590]
**Background (please complete the following information):**
* Panel or Daemon: Daemon (Wings)
* Version of Panel/Daemon: 1.0.0
* Server's OS: Debian 10
* Your Computer's OS & Browser: macOS 10.15.7 (19H2) / Firefox 81.0.2 (64-Bit)

**Describe the bug**
The timezone inside the docker container started by wings is always UTC. On my host machine, the time is correct:
```
$ date
Sa 24. Okt 15:45:41 CEST 2020

$ timedatectl
               Local time: Sa 2020-10-24 15:46:17 CEST
           Universal time: Sa 2020-10-24 13:46:17 UTC
                 RTC time: Sa 2020-10-24 13:46:18
                Time zone: Europe/Berlin (CEST, +0200)
System clock synchronized: yes
              NTP service: active
          RTC in local TZ: no

$ ls -l /etc/localtime
lrwxrwxrwx 1 root root 33 Okt 16 22:14 /etc/localtime -> /usr/share/zoneinfo/Europe/Berlin

$ cat /etc/timezone
Europe/Berlin
```

The time in the container also says "CEST", but it is off  by two hours. Also the `/etc/localtime` links to the wrong file. But `/etc/timezone` contains the correct value:
```
$ sudo docker exec 568abad0a082 date
Sat Oct 24 13:46:59 CEST 2020

$ sudo docker exec 568abad0a082 ls -l /etc/localtime
lrwxrwxrwx 1 root root 27 Jul 20 00:00 /etc/localtime -> /usr/share/zoneinfo/Etc/UTC

$ sudo docker exec 568abad0a082 cat /etc/timezone
Europe/Berlin
```

When running bash inside the container, it looks like the TZ environment variable isn't set properly as `echo $TZ` returns an empty string.

When doing `docker inspect` on a container, it is configured for the container. Also the mount for `/etc/localtime` seems to be set:
```
"Env": [
    "TZ=CEST",
    // [...]
],

"Mounts": [
    // [...]
    {
        "Type": "bind",
        "Source": "/etc/localtime",
        "Destination": "/etc/localtime",
        "Mode": "",
        "RW": false,
        "Propagation": "rprivate"
    },
    {
        "Type": "bind",
        "Source": "/etc/timezone",
        "Destination": "/etc/timezone",
        "Mode": "",
        "RW": false,
        "Propagation": "rprivate"
    }
],
```

<details>
  <summary>docker info output</summary>

  ```
Client:
 Debug Mode: false

Server:
 Containers: 3
  Running: 3
  Paused: 0
  Stopped: 0
 Images: 3
 Server Version: 19.03.13
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 8fba4e9a7d01810a393d5d25a3621dc101981175
 runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
 init version: fec3683
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 4.19.0-11-amd64
 Operating System: Debian GNU/Linux 10 (buster)
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 19.6GiB
 Name: [...]
 ID: [...]
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No swap limit support
  ```
</details>

**To Reproduce**
1. Change your timezone on your Node to Europe/Berlin (or any other timezone that's not UTC)
2. Create a server through the panel
3. Start the server
4. The timezone inside the container is wrong

**Expected behavior**
I expect Wings to set the correct timezone to the container and the container to use that.

**Comment by parkervcp** [comment]
Have you updated the panel to 1.0.2 and wings to 1.0.1 yet?

**Comment by GiovaniFerraroTrivelli** [comment]
I updated to the latest versions of panel and wings and I am having the same issue.

My timezone (host) is "America/Argentina/Buenos_Aires" (-03 UTC) but when I execute "date" in docker, it prints "Sat Oct 24 14:25:43 America 2020".

**Comment by tinyoverflow** [comment]
@parkervcp I have updated to Wings 1.0.1 in the meantime but it didn't resolve the issue. Updating to Panel 1.0.2 did. I'm not sure how the panel correlates to that, maybe I haven't realized that it got fixed with Wings 1.0.1 already.

I've updated using the following procedure:
1. Download Wings 1.0.1 and place it inside /usr/local/bin
2. I've restarted my wings system service

For the panel: I did as the guide said: https://pterodactyl.io/panel/1.0/upgrade/1.0.html
I won't close the issue by myself as other people seem to have the same issue even after upgrading.

**Comment by DaneEveritt** [comment]
Your panel upgrade had nothing to do with the timezone, I can promise you that. You have to restart servers after upgrading to get any changes that were made to Docker.

I specifically stopped bind mounting timezone information into containers, the Docker images themselves will need to be adjusted to handle `TZ` variables passed in from the host machine.
